keyboards/keebio/iris/keymaps/jhelvy/keymap.c

```diff
@@ -0,0 +1,54 @@
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_QWERTY] = LAYOUT( \
+                  KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                               KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_EQL, \
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                               KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_MINS, \
+        KC_BSPC, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                               KC_H,    KC_J,    KC_K,    KC_L,    KC_QUOT,  KC_ENT, \
+        KC_LSPO, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,     KC_NO,            KC_NO,  KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSPC, \
+                                    LALT(KC_A),KC_LGUI,    MO(1),                    KC_SPC,  MO(2),     KC_PGDN),
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
They actually serve a purpose for me. I export the .json file from the QMK Configurator, then use a simple R program (https://github.com/jhelvy/qmkJsonConverter) to convert the json to the keymap.c layout format and save it as a plain .txt file. I use the backslashes in the text parser to denote a new line.